scripts/antlr4/Cypher.g4

```diff
@@ -280,7 +280,7 @@ kU_AttachDatabase
     : ATTACH SP StringLiteral (SP AS SP oC_SchemaName)? SP '(' SP? DBTYPE SP oC_SymbolicName (SP? ',' SP? kU_Options)? SP? ')' ;
 
 kU_Option
-    : oC_SymbolicName SP? '=' SP? oC_Literal ;
+    : oC_SymbolicName (SP? '=' SP? | SP*) oC_Literal | oC_SymbolicName;
```

Comment:
Why is this grammar rule changing? It doesn't relate to what PR is trying to implement.

Reply:
I think this is from Sterling's commit:
https://github.com/kuzudb/kuzu/commit/7c69437d92083f9ad6fd6f8c21cda740dbca9984
I believe it's because he didn't update `scripts/antlr4/Cypher.g4` in his PR, so it got updated in mine.